DNN Platform/Modules/ResourceManager/Components/Common/Utils.cs

```diff
@@ -1,7 +1,6 @@
 ﻿// Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information
-
 namespace Dnn.Modules.ResourceManager.Components.Common
```

Comment:
Shouldn't this be `DotNetNuke.Modules.ResourceManager.*` for the namespace instead of `Dnn.Modules.ResourceManager.*`. I see this throughout the new module and to the best of my knowledge we agreed we are to use `DotNetNuke.*`

Reply:
That is a good question... It was there so I am unsure :)

Reply:
cc: @mitchelsellers @bdukes 

I think it would be appropriate for the folx working on the new module to update the namespace. They will have the most experience on fixing any problems that come up. Especially with web api routing issues where the frontend code is communicating with the `DnnApiController`